tests/property/ic_api/arg_data_raw/test/tests.ts

```diff
@@ -0,0 +1,109 @@
+                    async (
+                        initArgData,
+                        postUpgradeArgData,
+                        queryArgData,
+                        updateArgData
+                    ) => {
+                        execSync(
+                            `dfx canister uninstall-code canister || true`,
+                            {
+                                stdio: 'inherit'
+                            }
+                        );
+
+                        const initArgDataCandidString = `("${escapeArgData(
+                            initArgData
+                        )}")`;
+
+                        execSync(
+                            `dfx deploy canister --argument '${initArgDataCandidString}'`,
+                            {
+                                stdio: 'inherit'
+                            }
+                        );
+
+                        const actor = await getCanisterActor<Actor>('canister');
+                        const initArgDataRaw = await actor.getInitArgDataRaw();
+                        const expectedInitArgDataRaw = await actor.candidEncode(
+                            initArgDataCandidString
+                        );
+                        expect(initArgDataRaw).toEqual([
+                            expectedInitArgDataRaw
+                        ]);
+
+                        const queryArgDataCandidString = `("${escapeArgData(
+                            queryArgData
+                        )}")`;
+                        const queryArgDataRaw =
+                            await actor.getQueryArgDataRaw(queryArgData);
+                        const expectedQueryArgDataRaw =
+                            await actor.candidEncode(queryArgDataCandidString);
+                        expect(queryArgDataRaw).toEqual(
+                            expectedQueryArgDataRaw
+                        );
+
+                        const updateArgDataCandidString = `("${escapeArgData(
+                            updateArgData
+                        )}")`;
+                        const updateArgDataRaw =
+                            await actor.getUpdateArgDataRaw(updateArgData);
+                        const expectedUpdateArgDataRaw =
+                            await actor.candidEncode(updateArgDataCandidString);
+                        expect(updateArgDataRaw).toEqual(
+                            expectedUpdateArgDataRaw
+                        );
+
+                        const postUpgradeArgDataCandidString = `("${escapeArgData(
+                            postUpgradeArgData
+                        )}")`;
+
+                        execSync(
+                            `dfx deploy canister --argument '${postUpgradeArgDataCandidString}' --upgrade-unchanged`,
+                            {
+                                stdio: 'inherit'
+                            }
+                        );
+
+                        const postUpgradeArgDataRaw =
```

Comment:
During the init test and during the post upgrade test, let's test that the other method's arg data variable is None

Reply:
done